src/main/java/com/google/sps/servlets/startGameServlet.java

```diff
@@ -45,6 +45,11 @@ public startGameServlet(){
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
  
     String gameName = request.getParameter("game-name");
+
+    if(gameName == ""){
```

Comment:
Might want to change to gameName.isEmpty()

Reply:
Got it; changed.